src/or/entrynodes.c

```diff
@@ -1231,43 +1234,38 @@ entry_guard_is_listed,(guard_selection_t *gs, const entry_guard_t *guard))
   }
 }
 
-/**
- * Update the status of all sampled guards based on the arrival of a
- * new consensus networkstatus document.  This will include marking
- * some guards as listed or unlisted, and removing expired guards. */
-STATIC void
-sampled_guards_update_from_consensus(guard_selection_t *gs)
+/** Enumerate <b>sampled_entry_guards</b> smartlist in <b>gs</b>.
```

Comment:
Not exactly the right function documentation style. For multi-line comments perhaps leave the `/**` line empty. See how other functions do it.

Reply:
LGTM.